metals/src/main/scala/scala/meta/internal/worksheets/WorksheetProvider.scala

```diff
@@ -469,6 +467,7 @@ object WorksheetProvider {
   }
   final case class MdocRef(scalaVersion: String, value: Mdoc)
 
+  // TODO
```

Comment:
Leftover TODO?

Reply:
Oh, I forgot about it.
Yeah, it's for @tgodzik because he introduced these methods :D

Reply:
Wait, but what would I need to do?

Reply:
what is the difference between both `worksheetScala3Adjustments` if they both ignore the second parameter? Why one is adding +1 to the position?

Reply:
The second parameter looks like it can be removed. The second method has the additiona Position mapping. It's +1 because we wrap worksheet input in an object, since scripts do not yet work for Scala 3